internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -208,6 +208,9 @@ func (f *FBCRegistryPod) podForBundleRegistry(cs *v1alpha1.CatalogSource) (*core
 			Namespace: f.cfg.Namespace,
 		},
 		Spec: corev1.PodSpec{
+			SecurityContext: &corev1.PodSecurityContext{
+				RunAsNonRoot: &[]bool{true}[0],
```

Comment:
We might want to add a securityContext to the container as well:
```
securityContext:
    privileged:  false,
    readOnlyRootFilesystem: false,
    allowPrivilegeEscalation: false,
    capabilities:
       drop: ["ALL"]
```

Reply:
finally, I think to comply with the PSA `restricted` profile, you may need to specify the `seccompProfile` in the pod's security context. e.g.
```
securityContext:
    ...
    seccomProfile:
        type: RuntimeDefault
```

However, this would only work on k8s 1.19+